### DISCUSSION TITLE
how to insert data to list?

### DISCUSSION THREAD

**Discussion by ghost** [msiemens/tinydb#525] [Q&A] [answered]
can you please explain, how to add objects to list (add function)?
please, show me the example
thank you!

**Comment by msiemens** [comment, marked as answer]
Hey @astrofox09, I assume you ask this in context of the `db.update()` method? If yes, it'd go like this:

```python
db.insert({'foo': [{'name': 'bar'}]})
db.update(lambda doc: doc['foo'].append({'name': 'baz'}))
```

Does this help you?
